src/tools/cephfs/cephfs-shell

```diff
@@ -157,8 +157,13 @@ def get_all_possible_paths(pattern):
     for pattern in patterns:
         for path in paths:
             paths.extend(glob(path, pattern))
-    return [path for path in paths if fnmatch.fnmatch(path,
-            os.path.join(cephfs.getcwd(), complete_pattern))]
+    pat = ""
+    if is_rel_path:
+        parent_dir = cephfs.getcwd().decode('utf-8')
```

Comment:
Paths are now always treated as a bytes object. We never decode them unless printing to some external message.